python/ray/tune/logger/logger.py

```diff
@@ -2,7 +2,7 @@
 import json
 import logging
 
-from typing import TYPE_CHECKING, Dict, List, Optional, Type, Iterable
+from typing import TYPE_CHECKING, Dict, List, Optional, Type, Iterable, Set
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Dict, Iterable, List, Optional, Set, Type
```
nit alphabetical order